docs/topics/db/fixtures.txt

```diff
@@ -126,11 +126,31 @@ You could also write a decorator to encapsulate this logic::
 Just be aware that this logic will disable the signals whenever fixtures are
 deserialized, not just during ``loaddata``.
 
-Note that the order in which fixture files are processed is undefined. However,
-all fixture data is installed as a single transaction, so data in
-one fixture can reference data in another fixture. If the database backend
-supports row-level constraints, these constraints will be checked at the
-end of the transaction.
+Fixtures can be loaded in a specified order. For example:
+
+.. code-block:: shell
+
+    manage.py loaddata foo bar baz
+
+or (in a test case)
+
+.. code-block:: shell
+
+    Fixtures = ['foo', 'bar', 'baz']
+
+In this example, ``foo`` refers to any fixture named ``foo`` in any application.
+This would load ``foo`` followed by ``bar`` and then ``baz``.
+
+Where there are several fixtures with the same name, the order in which
+applications are defined in ``INSTALLED_APPS`` is also taken into account.
+The application listed first will be the one whose fixture is loaded first.
```

Comment:
Only after submitting the suggestion above, I read this paragraph. I think it would be idea to come up with a unified paragraphs with the combined information for fixtures order + app order.